scripts/common.sh

```diff
@@ -16,6 +16,8 @@ KUBECTL=$(which kubectl)
 export KUBECTL
 MINIKUBE=$(which minikube)
 export MINIKUBE
+KUSTOMIZE=$(which kustomize)
```

Comment:
Who ever said portable shell scripting was easy? https://unix.stackexchange.com/a/85250 

My take away is we should probably do 
```suggestion
KUSTOMIZE=$(command -v kustomize)
```

Though I'm happy to come back and do that in a follow up PR.

Reply:
Good catch, let's do that in a oner in a separate PR.